strict_encoding/src/monero.rs

```diff
@@ -0,0 +1,258 @@
+// LNP/BP client-side-validation foundation libraries implementing LNPBP
+// specifications & standards (LNPBP-4, 7, 8, 9, 42, 81)
+//
+// Written in 2019-2022 by
+//     Dr. Maxim Orlovsky <orlovsky@pandoracore.com>
+//     h4sh3d <h4sh3d@protonmail.com>
```

Comment:
It is just you :)

Reply:
Done in b5507de1eda92ac2ffae11064bf22034aed93f7f